File: TTs/TT02/Trabalho_Teórico_02/08_donaMonica_C/donaMonica.c
/* Bibliotecas Usadas*/
#include <stdlib.h>
#include <stdio.h>

/* MÉTODO PRINCIPAL */
int main(){
    int idadeM, idadeV, idadeN[2];
    scanf("%d", &idadeM);
    while(idadeM != 0){
        scanf("%d %d", &idadeN[0], &idadeN[1]);
        idadeV = idadeM - idadeN[0] - idadeN[1];

        if(idadeV > idadeN[0] && idadeV > idadeN[1]) printf("%d\n", idadeV);
        else if(idadeN[0] > idadeN[1]) printf("%d\n", idadeN[0]);
        else printf("%d\n", idadeN[1]);

        scanf("%d", &idadeM);
        }
    return 0;
}

File: TTs/TT02/Trabalho_Teórico_02/07_donaMonica/donaMonica.java
public class donaMonica {

    
    public static void main(String[] args){
        int idadeM = MyIO.readInt(),
            idadeV = 0;
        int[] idadeN = new int[2];

        while(idadeM != 0){
            idadeN[0] = MyIO.readInt();
            idadeN[1] = MyIO.readInt();

            idadeV = idadeM - (idadeN[0] + idadeN[1]);
            if(idadeV > idadeN[0] && idadeV > idadeN[1]) System.out.println(idadeV);
            else if(idadeN[0] > idadeN[1]) System.out.println(idadeN[0]);
            else System.out.println(idadeN[1]);

            idadeM = MyIO.readInt();
        }
    }
}
